src/webapp/app.py

```diff
@@ -260,20 +260,15 @@ def graph():
 
         # <ms_docref_acquire_token>
         # Invoke the acquire_token flow on the MSAL client for the requested
-        # scope and account. This will use the cache to either retrieve an
-        # existing valid token or will use the refresh token in the cache to
-        # fetch a new access token. A complete cache miss will result in an
-        # error and indicates that a former auth code flow exchange didn't
-        # include this scope in the request.
+        # scope and account. Look for and retrieve an existing valid token in
```

Comment:
account and scope. Look for and retrieve an existing valid token in

Reply:
Thanks @henrymbuguakiarie, I've made this update